src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ShowColumns.txt

```diff
@@ -0,0 +1,31 @@
+﻿#SETUP: SupportColumnNamesAsIdentifiers
+
+>> ShowColumns(Table({a:1,b:2},{a:3,b:4}), b)
+Table({b:2},{b:4})
+
+>> ShowColumns(Table({a:1,b:2},{a:3,b:4}), b, a)
+Table({a:1,b:2},{a:3,b:4})
+
+>> ShowColumns(ShowColumns(Table({a:1,b:2},{a:3,b:4}), a), a)
+Table({a:1},{a:3})
+
+>> ShowColumns(AddColumns(Table({a:1,b:2},{a:3,b:4}), c, a * b), c)
+Table({c:2},{c:12})
+
+>> AddColumns(ShowColumns(Table({a:1,b:2},{a:3,b:4}), a), c, a * a * a)
```

Comment:
>AddColumns(ShowColumns(Table({a:1,b:2},{a:3,b:4}), a), c, a * a * a) [](http://example.com/codeflow?start=3&length=68)

It would be good to have a test where we add a column that has been removed, something like `AddColumns(ShowColumns(Table({a:1,b:2},{a:3,b:4}), a), b, a * a * a)`